src/frontend/hooks/useJaNeiSpmFelt.tsx

```diff
@@ -40,20 +42,25 @@ export const erRelevanteAvhengigheterValidert = (avhengigheter: { [key: string]:
 
 const useJaNeiSpmFelt = ({
     søknadsfelt,
+    feilmelding,
     feilmeldingSpråkId,
     avhengigheter,
     nullstillVedAvhengighetEndring = false,
     skalSkjules = false,
+    flettefelter,
     feilmeldingSpråkVerdier,
 }: {
     søknadsfelt?: ISøknadSpørsmål<ESvar | null>;
+    feilmelding?: LocaleRecordBlock;
     feilmeldingSpråkId: string;
```

Comment:
Burde `feilmeldingSpråkId` bli optional? Så kan du fjerne disse dette feltet ettersom det ikke brukes på de oppdaterte stedene 😃

Reply:
Jeg er litt redd for at dersom både `feilmelding` og `feilmeldingSpråkId` er optional, kan noen legge til et `JaNeiSpm` uten verken av dem, som blir feil.

Planen min var å legge til `feilmelding` frem til alle `feilmeldingSpråkId` ikke lengre tas i bruk og deretter fjerne `feilmeldingSpråkId` helt, hva tenker du om det?

Reply:
Jeg og @ole-kristian-rudjord har snakket om det. Det kan føre til feilsituasjoner hvis begge feltene er optional, dermed lar vi `feilmeldingSpråkId` være obligatorisk enn så lenge. Så fjerner vi det helt når alle spørsmål er over.